src/FurCoNZ.Web/Areas/Debug/Controllers/HomeController.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+using FurCoNZ.Web.DAL;
+using FurCoNZ.Web.Models;
+using FurCoNZ.Web.Services;
+
+namespace FurCoNZ.Web.Areas.Debug.Controllers
```

Comment:
Shouldn't this code be inside of 
```c#
#if DEBUG
...
#endif
```

To really make sure the code doesn't make it's way into production or Release builds